src/components/LikeButton/LikeButton.st.css

```diff
@@ -0,0 +1,69 @@
+:import {
+    -st-from: "../../../src/tpaStyles/styleableExt";
+    -st-named: wixApply;
+}
+
+:import {
+    -st-from: "../IconToggle/IconToggle.st.css";
+    -st-default: IconToggle;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: wixApply(color, color-5);
+    CheckedIconColor: '#F64D43';
+    DefaultFont: wixApply(font, "{theme: 'Body-M', size: '12px', lineHeight: '16px'}");
+}
+
+:vars {
+    /*Overrides*/
+    IconColor: --overridable;
+    LabelColor: --overridable;
+    LabelFont: --overridable;
+}
+
+:vars {
+    iconColor: wixApply(color, wixApply(fallback, value(IconColor), value(DefaultColor)));
+    checkedIconColor: value(CheckedIconColor);
+    labelColor: wixApply(color, wixApply(fallback, value(LabelColor), value(DefaultColor)));
+    labelFont: wixApply(font, wixApply(fallback, value(LabelFont), value(DefaultFont)));
+}
+
+.root {
+    -st-states: checked;
+    -st-extends: IconToggle;
+    -st-mixin: IconToggle(
+        IconColor value(iconColor),
+        LabelColor value(labelColor),
+        LabelFont value(labelFont)
+    );
+}
+
+.root:checked {
+    -st-mixin: IconToggle(
+        IconColor value(checkedIconColor),
+        LabelColor value(labelColor),
+        LabelFont value(labelFont)
+    );
+}
+ 
+@keyframes pop-in {
+    0% {
+        transform: scale(0.9);
+        opacity: 0;
+    }
+    50% {
+        transform: scale(1.03);
+        opacity: 0.32;
+    }
+    100% {
+        transform: scale(1);
+        opacity: 1;
+    }
+}
+
+.root:checked::icon path {
+    -webkit-animation: pop-in 0.2s ease 0s 1 normal ;
+   animation: pop-in 0.2s ease 0s 1 normal ; 
+   animation-fill-mode: forwards;
```

Comment:
Nice work with the animation!
Just one comment: I see the the animation is being done on the `path`, and not on the `.icon` itself.
This isn't a problem on itself, but notice that the `hover` states affect the `.icon` and not the `path`.
This means we don't get to see the bright red when clicking on the heart icon, because the icon has `opacity: 0.6`